lib/roken/version-script.mapin

```diff
@@ -229,13 +309,9 @@ HEIMDAL_ROKEN_2.0 {
 		unparse_time_approx;
 		unparse_units;
 		unparse_units_approx;
-		verr;
```

Comment:
We don't even build `lib/roken/err.c` on non-Windows, so maybe we should also not refer to `err.h` in `lib/roken/Makefile.am`.  (NOTE: I'm not asking OP to make any change in this respect.)